S21/jackzhenguo/refactorme.py

```diff
@@ -1,84 +1,112 @@
 import unicodedata
 
+
 class Word:
+    def __init__(self, word, bcp_code):
+        self.wc = word
+        self.lang = bcp_code
+
+
+class Strategy(object):
```

Comment:
I'd suggest "LowercaseStrategy" for this.

Reply:
You're tying this to the lowercasing application by including the Word as an attribute. If you want to preserve the generic strategy interface, you'd need that as a parent of LowercaseStrategy, with no attribute "word".